### ISSUE TITLE
Peer certificate cannot be authenticated with given CA certificates

### ISSUE THREAD

**Issue by mtmorgan** [Bioconductor/BiocManager#32]
https://support.bioconductor.org/p/114876/#114914 shows that accessing the config.yaml is via https://, and that this is problematic for some users.

I'd rather fix the user than provide a way to use http:// ? So what is the solution to authenticating our certificate? This is on Linux. A similar problem seems to be issue #30, on Windows.

**Comment by mtmorgan** [comment]
A workaround might be to implement `getOption("useHTTPS", FALSE)`, which is how BiocInstaller worked (actually BIocInstaller tried to guess at http / https; I really think we should move our users away from http as forcefully as possible).

**Comment by mtmorgan** [comment]
A patch allowing fallback to http is at https://github.com/Bioconductor/BiocManager/tree/feature/http-fallback ; install with `remotes::install_github("Bioconductor/BiocManager", ref="feature/http-fallback")`. It likely requires that Bioconductor (and perhaps CRAN) packages are installed using http
```
options(BioC_mirror = "http://bioconductor.org")  # perhaps also 'repos="http://cran.r-project.org")
```

**Comment by Sanchezillana** [comment]
I have not been able to install fallback. First, I have installed the "remotes" package employing http:// CRAN server (selecting the option in the RStudio menu) but for fallback the problem is in https://api.github.com/...

Then, 
> options(BioC_mirror = "http://bioconductor.org")
> remotes::install_github("Bioconductor/BiocManager", ref="feature/http-fallback")
Error in utils::download.file(url, path, method = download_method(), quiet = quiet,  : 
  cannot open URL 'https://api.github.com/repos/Bioconductor/BiocManager/commits/feature%2Fhttp-fallback'

I think that if the problem is with the https it may solved employing the http for the GitHub in "remotes" as well but I do not know how to do it.

**Comment by mtmorgan** [comment]
I think that this is a symptom of the same problem on your computer, that the security certificates are not configured properly, or perhaps that you are behind a firewall and need to use a proxy.

You should really fix this, but it is beyond the help that I can provide; speak with your local computer support person.

You could visit https://github.com/Bioconductor/BiocManager/tree/feature/http-fallback , click on the 'clone or download' button, choose 'Download zip'. 'unzip' the file by clicking on it, and then in R, use install.packages locally `install.packages("path/to/BiocManager-feature-http-fallback", repos=NULL)`.

**Comment by mtmorgan** [comment]
This is fixed in 0778b51 and made available in 1.30.4, falling back to http.

**Comment by vanhoan310** [comment]
I got the same problem because I use R on conda environment. I fixed it add file "cacert.pem" (in the anaconda environment) to R as follow.
Sys.setenv(CURL_CA_BUNDLE = "//path//to//cacert.pem file")
For details please see in here: http://tinyheero.github.io/2016/11/12/bioconductor-conda.html
